### DISCUSSION TITLE
q-uploader addFiles Example

### DISCUSSION THREAD

**Discussion by doornema** [quasarframework/quasar#14365] [General - Components / Directives / etc]
hi
i want download images from server and add to uploader 
this is my code but uploader.addfiles is not working 
image show truly but not add to uploader
api
        .get(`/static/avatar/default_avatar.jpg`, {
          method: "GET",
          responseType: "blob",
        })
        .then((r) => {
          console.log("downloaded");

          const image= document.getElementById("img");
          var reader = new window.FileReader();
          reader.readAsDataURL(r.data);
          reader.onload = function () {
            var imageDataUrl = reader.result;
            image.setAttribute("src", imageDataUrl);
            uploader.value.addFiles([imageDataUrl]);
          };
        })
        .catch((e) => {
          console.log(e);
        });

???

**Comment by MilosPaunovic** [comment]
Please provide a code reproduction of some sort.
